.sun-ci.yml

```
@@ -0,0 +1,90 @@
+default:
+  workspace: shared
+
+stages:
+  - build
+  - check
+
+jobs:
+  - name: Build Laravel
+    stage: build
+    image: ghcr.io/avengerscodelovers/php:8.2-ci
+    services:
+      - image: mysql:8.0
+        name: mysql
+        environment:
+          MYSQL_DATABASE: app_db
+          MYSQL_USER: app_user
+          MYSQL_PASSWORD: secret
+          MYSQL_ROOT_PASSWORD: root
+    environment:
+      APP_ENV: testing
+      COMPOSER_MEMORY_LIMIT: 2G
+      COMPOSER_ALLOW_SUPERUSER: "1"
+    cache:
+      - key: composer_vendor_$CI_BRANCH
+        paths:
+          - vendor
+    script:
+      - cp .env.testing.example .env
```

ở [trên](https://github.com/AvengersCodeLovers/ci-laravel/pull/1/files#diff-b47c1605e9037d40a624e25a0cedc8335244491d9fe56751767b51338c49b1c6R21) định nghĩa `APP_ENV: testing` rồi thì ở đây dùng file `.env.testing` cho [khớp](https://laravel.com/docs/11.x/configuration#additional-environment-files) nhờ
```suggestion
      - cp .env.testing.example .env.testing
```